x-pack/solutions/observability/plugins/slo/server/services/slo_repository.ts

```diff
@@ -32,11 +32,12 @@ export interface SLORepository {
 export class KibanaSavedObjectsSLORepository implements SLORepository {
   constructor(private soClient: SavedObjectsClientContract, private logger: Logger) {}
 
-  async exists(id: string) {
+  async exists(id: string, namespaces = []) {
     const findResponse = await this.soClient.find<StoredSLODefinition>({
       type: SO_SLO_TYPE,
       perPage: 0,
       filter: `slo.attributes.id:(${id})`,
+      namespaces: [...namespaces],
```

Comment:
this will only work to spaces which user have access to. 
you need to use kibana internal user for this kinda use-case.